vuegridlayout.css

```diff
@@ -23,6 +23,11 @@ div#layoutApp .layoutJSON {
   margin: 0;
 }
 
+.layoutJSON button.current-tableau-button {
+  background: yellow !important;
```

Comment:
I don't think you need `!important` here, and therefore you should remove it.
Your `.layoutJSON button.current-tableau-button` selector is *more specific* than the Bootstrap style you are *competing with*, so it has precedence. This is because it is composed of 3 selectors `.layoutJSON`, `button`, and `current-tableau-button` and Bootstrap's class is a single selector `.btn-default`.

See https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity